hardhat.config.ts

```diff
@@ -1,16 +1,19 @@
 import * as dotenv from 'dotenv';
 import { HardhatUserConfig } from 'hardhat/config';
 import '@nomicfoundation/hardhat-toolbox';
+import '@nomicfoundation/hardhat-foundry';
 
 import 'tsconfig-paths/register';
 import { ChainId } from './constants';
 
 dotenv.config();
 
-const { PRIVATE_KEY, ETHERSCAN_API_KEY, POLYGONSCAN_API_KEY, DEPLOYER_PRIVATE_KEY, REPORT_GAS, INFURA_API_KEY } = process.env;
+const { PRIVATE_KEY, ETHERSCAN_API_KEY, POLYGONSCAN_API_KEY, REPORT_GAS, INFURA_API_KEY } = process.env;
 
-if (!PRIVATE_KEY) {
-  throw new Error('The private key is required');
+let accounts: string[] = [];
+const privateKey: string | undefined = PRIVATE_KEY;
+if (privateKey) {
+  accounts = [privateKey];
 }
 
 const config: HardhatUserConfig = {
```

Comment:
unrelated but while you're here Ganache is deprecated , so it seems like our local host shoudl default to the standard 8545, with maybe the chainId 31337 
```
      url: "http://127.0.0.1:8545",
      chainId: 31337
```

Reply:
rm the accounts entry as well.

Reply:
I think the local development setup is using ganache, so don't want to edit this yet until we have instructions for how to bring up a persistent blockchain using Hardhat network.